### ISSUE TITLE
Improvement for k8s.io/docs/setup/pick-right-solution/

### ISSUE THREAD

**Issue by triesz** [kubernetes/website#12364]
**This is a...** 

- [x] Feature Request
- [ ] Bug Report

**Problem:**
I wanted to find out what is the best way to install a small kubernetes "playground" cluster and start learning on 3 low end stick PCs.

This is the very first page of the doc and I found it very overwhelming and in fact it did not really help at all to find the answer. Instead it was super time consuming to scan through all these solutions and links...

**Proposed Solution:**
Is it a valid assumption that coreos is the most performance efficient way to host kubernetes on a bare machine? If so could we perhaps explicitly point this out at the "Bare Metal" section?

Basically the info I needed and I did not find was:
1) install coreos manually on your machines
2) continue with the "Bootstrapping Clusters with kubeadm" section

(I only accidentally discovered after more than a day that the setup guide is practically the "Bootstrapping Clusters with kubeadm" part and that there is a dedicated tab for coreos there!)

Instead I got lost on the "Picking the Right Solution" page, going back and forth among all these links. (I.e. please note that the Bare metal -> Coreos links point out to docs using various cloud services...)

**Page to Update:**
https://kubernetes.io/...

**Comment by neolit123** [comment]
> This is the very first page of the doc and I found it very overwhelming and in fact it did not really help at all to find the answer. Instead it was super time consuming to scan through all these solutions and links...

even if the page is mostly unmaintained i don't think it's badly structured:
there is a kubeadm (Custom) and minikube links at the beginning.

> Is it a valid assumption that coreos is the most performance efficient way to host kubernetes on a bare machine?

there should be no such claims in the official k8s docs.
is that the case?

> install coreos manually on your machines

this is slightly out of scope for the kubernetes docs.
coreos has official install docs that we don't maintain.
we don't do that for any Linux distribution.

> continue with the "Bootstrapping Clusters with kubeadm" section

https://kubernetes.io/docs/setup/pick-right-solution/#custom-solutions
this page already contains a link to kubeadm in this sentance:
`If you already have a way to configure hosting resources, use kubeadm to easily bring up a cluster with a single command per machine`

/sig cluster-lifecycle

**Comment by triesz** [comment]
> > Is it a valid assumption that coreos is the most performance efficient way to host kubernetes on a bare machine?
> 
> there should be no such claims in the official k8s docs.
> isn't that the case?

Well, if someone would like to deploy and try a k. cluster on real HW then the very first question one needs to answer is what OS is reasonable to host it?
And I have not really found information on this topic here. Perhaps we could at least list some tested and working options... Maybe even compare kubernetes performance on a reference HW using different OS versions...

For example I saw in the doc
```
snap install conjure-up --classic
conjure-up kubernetes
```
So I created ubuntu installer, flashed ubuntu core on my stick PC only to learn that this way does not work on core, only on a classic system...

So a bit more info on what OS is tried and works OK and what is recommended would be very beneficial, IMO

**Comment by neolit123** [comment]
we recently did a survey where results showed that Ubuntu is the most popular node OS / distro.
16.04 is known to work pretty well, but i'd recommend something newer like 17.x

from there you can use kubeadm with these guides:
https://kubernetes.io/docs/setup/independent/
https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/
